KubeArmor/BPF/shared.h

```diff
@@ -403,6 +413,17 @@ static __always_inline u32 init_context(event *event_data) {
       event_data->exec_id = *exec_id;
   }
 
+  // check if tty is attached
+  struct signal_struct *signal;
+  signal = READ_KERN(task->signal);
+  if (signal != NULL){
+      struct tty_struct *tty = READ_KERN(signal->tty);
+      if (tty != NULL){
+          // a tty is attached
+          bpf_probe_read_str(&event_data->tty, TTY_LEN, (void *)tty->name);
```

Comment:
let's add this too.
   // to avoid garbage values
    __builtin_memset(event_data>tty, 0, sizeof(event_data->tty));